docs/search.md

```diff
@@ -0,0 +1,5 @@
+<form action="search.html">
```

Comment:
What is this file for? Is it elasticlunr specific?

This shouldn't be in the default template as it shouldn't be tied to a specific library but allow the users to do themselves whatever they need. For example, this could be in the user `docs` folder as `search.md` or can be a `search.html` on your output folder.